### DISCUSSION TITLE
The GPU usage is 0. Procedure，  need help

### DISCUSSION THREAD

**Discussion by 1hexf1** [PaddlePaddle/PaddleOCR#12935] [Q&A]
def ocr_det(image_path_list):
    ocr_model = PaddleOCR(use_angle_cls=True,lang='en', use_gpu=True, det=True, rec=False, show_log=False,use_tensorrt=True)
    print("ocr model done")
    num_det_list = []
    for image_ind, image_path in enumerate(image_path_list):
        if image_ind%100 == 0:
            print("ocr: %.8d/%d images"%(image_ind, len(image_path_list)))
        try:
            det_res = ocr_model.ocr(image_path, cls=True)
            num_det_list.append(len(det_res[0]))
        except:
            num_det_list.append(-1)
    return num_det_list

**Comment by g39088902** [comment]
Please prove that you have the cuda version of the paddlepaddle framework installed by using the "pip list | grep paddle" command
